kuma/users/jinja2/socialaccount/signup.html

```diff
@@ -1,212 +1,148 @@
-{% extends "react_base.html" %}
+{% extends "socialaccount/base_signup.html" %}
 
 {% set styles = ('users',) %}
-
-{% block breadcrumbs %}{% endblock %}
-
-{% set title = _('Create your MDN profile') %}
-
 {% set classes = 'register' %}
 
-{% block site_css %}
+{% block page_css %}
   {{ super() }}
-  {% stylesheet 'submission' %}
   {% stylesheet 'signupflow' %}
 {% endblock %}
 
 {% set provider = account.get_provider() %}
 {% set github_connect_url = provider_login_url('github', process='connect', next=request.session.sociallogin_next_url|default('/')) %}
 
 {% block pageheader %}
-  {{ render_react("signupflow", request.LANGUAGE_CODE, request.get_full_path(),
+{{ render_react("signupflow", request.LANGUAGE_CODE, request.get_full_path(),
```

Comment:
@schalkneethling Just an observation that I've been slow to realize, and there is nothing wrong with what you have here, but we really don't need to use React for the signup flow page(s) since their endpoints (e.g., `/en-US/users/account/signup`) are not and will never be cached by the CDN and are passed the session cookie. In other words, those endpoints could still continue to use the same Jinja templates we use for the Wiki endpoints (those templates use `request.user` for example rather than getting the user information with an AJAX call to the whoami endpoint).

Reply:
I believe the idea is that all of this(the entire site really) will eventually be rewritten in React and so, this is why I started to bring in the header here. Are you thinking we should stick to Jinja and friends for things that are behind a login and thus will never be cached?

Reply:
I'm not sure what I think yet. I just wanted to get that thought out in the open to see what you and the others think. It's probably still best to continue down the path of converting everything to React.